clar2wasm/tests/wasm-generation/sequences.rs

```diff
@@ -114,6 +114,59 @@ proptest! {
     }
 }
 
+proptest! {
+    #![proptest_config(super::runtime_config())]
+
+    #[test]
+    fn crosscheck_map_not(
+        seq in (0usize..=20).prop_flat_map(PropValue::any_sequence),
+        val in any::<u32>()
+    ) {
+        let expected = !extract_sequence(seq.clone()).len() > val as usize;
+        let snippet = format!("(map not (list (> (len {seq}) u{val})))");
```

Comment:
Still not convinced by that snippet. Why the difficult operation when you could directly generate a list of booleans? Why not try to map on lists of different lengths?

Reply:
Changed for an implementation that uses a boolean list of different sizes.